### ISSUE TITLE
can't create RRD file

### ISSUE THREAD

**Issue by paradisewendy** [scline/CactiTemplate_JuniperNetworks#4]
RRDTool can't create RRD file and debug error as below
RRDTool Says:
ERROR: opening '/usr/share/cacti/rra/63/50195.rrd': No such file or directory

**Comment by scline** [comment]
This is a problem with your cacti setup, not the templates on this repo. I
would advise verifying that directory exists and is writable by your cacti
user/installation.

The Cacti Forums http://forums.cacti.net/ may be of some assistance for
this issue.

**Comment by paradisewendy** [comment]
Thank you for your quick response. we did research via forums and still not able to find the root issue. the directory exist and cacti has write permission for that direction and snmp query looks ok via Cacti. We also did debugs on Poller and i couldn't find OID we have use.  Cacti is working for all other routers and other template and i couldn't find the root issue.

 -data query debug example
- Executing SNMP walk for list of indexes @ '.1.3.6.1.4.1.2636.3.5.2.1.8' Index Count: 33
- Index found at OID: '.1.3.6.1.4.1.2636.3.5.2.1.8.3.114.112.102.21.114.112.102.45.100.105.115.99.97.114.100.101.100.45.112.97.99.107.101.116.115.2' value: '2'
- Index found at OID: '.1.3.6.1.4.1.2636.3.5.2.1.8.8.108.111.48.45.105.112.118.54.25.115.109.97.108.108.45.98.119.45.108.105.109.105.116.45.97.108.108.111.119.45.105.99.109.112.3' value: '3'
- Index found at OID: '.1.3.6.1.4.1.2636.3.5.2.1.8.8.108.111.111.112.98.97.99.107.25.115.109.97.108.108.45.98.119.45.108.105.109.105.116.45.97.108.108.111.119.45.105.99.109.112.3' value: '3'

-graph debug mode example
RRDTool Says:
ERROR: opening '/usr/share/cacti/rra/63/50189.rrd': No such file or directory

-directory permission example. cacti has read write permission for our directory 63. but RRD file 
-rw-r--r--   1 cacti cacti 196760 Jun  2 10:10 50195.rrd
xxxxx:/usr/share/cacti/rra/63$ 
xxxxx:/usr/share/cacti/rra/63$ 
xxxxx:/usr/share/cacti/rra/63$ ls -la | grep 50189
xxxxx:/usr/share/cacti/rra/63$ 

-conclusion:
RRD file is not created even the permission is allowed. no error in cacti log so far
